recipes/pep8/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: pep8
```

Comment:
@pmlandwehr do what is easier for you. I don't like to fret over git :wink: 

PS: @jjhelmus it might be nice to get `pep8` merged though and then add `pycodestyle`, no?